WORKING_GROUPS.md

```diff
@@ -0,0 +1,296 @@
+# io.js Working Groups
+
+io.js Working Groups are autonomous projects created by the TC.
```

Comment:
Perhaps it would be good to newcomers (or people who just easily forget, like me) to make it a habit of writing out something like [Technical Committee (TC)] (https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#technical-committee) the first time we mention TC in docs?